src/core/app_backend.tf

```diff
@@ -717,7 +717,7 @@ module "appservice_app_backendl1_slot_staging" {
   ip_restriction_default_action = "Deny"
 
   allowed_subnets = [
-    module.azdoa_snet[0].id,
+    data.azurerm_subnet.azdoa_snet.id,
```

Comment:
do we still need to let the devops agent access io-backend?

Reply:
It looks like io-backend is still deployed via Azure DevOps and no private endpoint is present on the machines. I think we should keep this rule